File: src/service/ModelBuilder.Models/Data/Attribute.cs
using System;

namespace Mb.Models.Data;

public class Attribute 
{
    public Guid Id { get; set; }
    public string Name { get; set; }
    public string Value { get; set; }
    public string AttributeType { get; set; }
    public string UnitSelected { get; set; }
    public string Units { get; set; }
    public string Qualifiers { get; set; }
    public string Terminal { get; set; }
    public Guid? Block { get; set; }


}

File: src/client/src/lib/clsses/Attribute.ts
import { Datum, Unit } from ".";

export class Attribute {
  id: string;
  name: string;
  value: string;
  attributeType: string;
  isLocked: boolean;
  IsLockedStatusBy: string;
  IsLockedStatusDate: Date;
  selectedUnit: string;
  datums: Datum[];
  units: Unit[];
  terminal: string;
  aspectObject: string;
}
